Examples/WriteApiExample.cs

```diff
@@ -0,0 +1,330 @@
+                .Build();
+
+            //
+            // Write Data
+            //
+            using (var writeApi = influxDbClient.GetWriteApi(options))
+            {
+                //
+                // Handle the Events 
+                //
+                writeApi.EventHandler += (sender, eventArgs) =>
+                {
+                    switch (eventArgs)
+                    {
+                        // success response from server
+                        case WriteSuccessEvent successEvent:
+                            Console.WriteLine("WriteSuccessEvent: point was successfully written to InfluxDB");
+                            Console.WriteLine($"  - {successEvent.LineProtocol}");
+                            Console.WriteLine(
+                                "-----------------------------------------------------------------------");
+                            break;
+
+                        // unhandled exception from server
+                        case WriteErrorEvent errorEvent:
+                            Console.WriteLine($"WriteErrorEvent: {errorEvent.Exception.Message}");
+                            Console.WriteLine($"  - {errorEvent.LineProtocol}");
+                            Console.WriteLine(
+                                "-----------------------------------------------------------------------");
+                            break;
+
+                        // retrievable error from server
+                        case WriteRetriableErrorEvent error:
+                            Console.WriteLine($"WriteErrorEvent: {error.Exception.Message}");
+                            Console.WriteLine($"  - {error.LineProtocol}");
+                            Console.WriteLine(
+                                "-----------------------------------------------------------------------");
+                            break;
+
+                        // runtime exception in background batch processing
+                        case WriteRuntimeExceptionEvent error:
+                            Console.WriteLine($"WriteRuntimeExceptionEvent: {error.Exception.Message}");
+                            Console.WriteLine(
+                                "-----------------------------------------------------------------------");
+                            break;
+                    }
+                };
+
+                //
+                // Write by LineProtocol
+                //
+                writeApi.WriteRecord("influxPoint,writeType=lineProtocol value=11.11" +
+                                     $" {DateTime.UtcNow.Subtract(EpochStart).Ticks * 100}", WritePrecision.Ns,
+                    "my-bucket", "my-org");
+
+                //
+                // Write by LineProtocol - bad timestamp
+                //
+                writeApi.WriteRecord("influxPoint,writeType=lineProtocol value=11.11" +
+                                     $" {DateTime.UtcNow}", WritePrecision.Ns, "my-bucket", "my-org");
```

Comment:
Please use interpolation instead of concatenation:

```suggestion
                writeApi.WriteRecord($"influxPoint,writeType=lineProtocol value=11.11 {DateTime.UtcNow}", 
                    WritePrecision.Ns, "my-bucket", "my-org");
```